app/actions/websocket/teams.ts

```diff
@@ -52,3 +54,62 @@ export async function handleLeaveTeamEvent(serverUrl: string, msg: WebSocketMess
         }
     }
 }
+
+export async function handleUpdateTeamEvent(serverUrl: string, msg: WebSocketMessage) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
+    }
+
+    try {
+        const team = JSON.parse(msg.data.team) as Team;
+        database.operator.handleTeam({
+            teams: [team],
+            prepareRecordsOnly: false,
+        });
+    } catch {
+        // Do nothing
```

Comment:
shouldn't we log the error message?

Reply:
When I did this PR, I was following the previous WS handler patterns.  I haven't used and am not opposed to logging the error. @enahum what are your thoughts and could you share a logging example for RN?

Reply:
every console.log is removed from the production code, so it will do nothing until we add a logging framework

Reply:
How about I create a ticket to handle all `// Do nothing` catch comments for websockets? I don't see a logging framework ticket in Jira. I can create that one also and make it a dependency on for catching all the WS error catching code

Reply:
The logging franework is probably going to be started at some point in Q3 as well as the telemetry framework, this is in the roadmap for the platform team.

The ticket already exist https://mattermost.atlassian.net/browse/MM-22378